packages/pigment-css-react/src/processors/useTheme.ts

```diff
@@ -0,0 +1,64 @@
+import type { Expression } from '@babel/types';
+import { validateParams, type Params, type TailProcessorParams } from '@wyw-in-js/processor-utils';
+import { type Replacements, type Rules } from '@wyw-in-js/shared';
+import BaseProcessor from './base-processor';
+
+export type Primitive = string | number | boolean | null | undefined;
+
+export type TemplateCallback = (params: Record<string, unknown> | undefined) => string | number;
+
+export class UseThemeProcessor extends BaseProcessor {
```

Comment:
I thought that we were going with option B from https://www.notion.so/mui-org/material-ui-pigment-non-processor-replacement-0c7e4b150c90499b93cd8fc17c31a009?pvs=4

Reply:
I realised that tagProcessor handles all the different kinds of imports that we can have, ie,

```js
import * as lib from 'lib';
import {useTheme} from 'lib';
require('lib')
interopRequire(require('lib'));
```

Doing it ourselves might miss something especially for library code where most of the code is transpiled.